### PR TITLE
feat (dynamic-sampling): Add dynamicSamplingRules to project details endpoint [TET-582]

### PR DESCRIPTION
**Pull request by andriisoldatenko**
This PR adds `dynamicSamplingRules` to project details endpoint for all `staff users` if you add `includeDynamicSamplingRules=1` option to query.

Example:

If user selects only 1 bias (dev env):

image

We show only 2 rules dev env bias and uniform sample rule:

GET http://localhost:8000/api/0/projects/sentry/earth/?includeDynamicSamplingRules=1

image

And we don't include rules without query param: `GET http://localhost:8000/api/0/projects/sentry/earth/`

image

### CODE REVIEW

File: src/sentry/quotas/base.py [hunk 1]
```diff
@@ -469,3 +469,4 @@ def get_blended_sample_rate(self, project: "Project") -> Optional[float]:
 
         :param project: The project model.
         """
+        return 0.5
```
Comment (iambriccardo) [inline src/sentry/quotas/base.py:472]: Why do we have the `get_blended_sample_rate` defined here?

Comment (andriisoldatenko) [reply src/sentry/quotas/base.py:472]: ooops :)

**Review by iambriccardo — APPROVED** [review]
LGTM

File: src/sentry/quotas/base.py (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/quotas/base.py]: testing :p

File: tests/sentry/api/endpoints/test_project_details.py [hunk 2]
```diff
@@ -1188,6 +1188,69 @@ def setUp(self):
...
+                self.new_ds_flag: True,
+            }
+        ):
+            response = self.get_success_response(
+                self.organization.slug,
+                self.project.slug,
+                method="get",
+                includeDynamicSamplingRules=1,
+            )
+            assert len(response.data["dynamicSamplingRules"]) == 3
```
Comment (priscilawebdev) [inline tests/sentry/api/endpoints/test_project_details.py:1220]: shall we set his information in project before?

File: tests/sentry/api/endpoints/test_project_details.py [hunk 3]
```diff
@@ -1188,6 +1188,69 @@ def setUp(self):
...
+            {
+                self.new_ds_flag: False,
+            }
+        ):
+            response = self.get_success_response(
+                self.organization.slug, self.project.slug, method="get"
+            )
+            assert response.data["dynamicSamplingRules"] is None
+
+    def test_non_stuff_user(self):
```
Comment (priscilawebdev) [inline tests/sentry/api/endpoints/test_project_details.py:1233]:
```suggestion
    def test_non_stuff_user_trying_to_access_dynamic_sampling_rules(self):
```

🤷‍♀️

**Comment by priscilawebdev** [comment]
Amazing PR description 🎉 thanks for adding it

**Review by priscilawebdev — APPROVED** [review]
code lgtm

**Review by jjbayer — APPROVED** [review]

**Pull request merged**